app/views/subscription/index.phtml

```diff
@@ -33,8 +33,13 @@
 				$feeds = $cat->feeds();
 		?>
 		<div class="box">
+			<?php
+				$categoryDAO = FreshRSS_Factory::createCategoryDao();
+				$category = $categoryDAO->searchById($cat->id());
```

Comment:
We probably already have all the categories in our context. See also `$this->categories` and its definition in:

https://github.com/FreshRSS/FreshRSS/blob/3b54f8cd8196728df14590bfb1560dd4f6aaf685/app/Controllers/subscriptionController.php#L22

Ups, by the way, I can see that it looks like we have one `true` too many

Reply:
I fixed the doubled parameter